### ISSUE TITLE
Task /admin: Enhance login page with Error Handling and UX Improvements

### ISSUE THREAD

**Issue by webcrumbs-community** [webcrumbs-community/webcrumbs#156]

**Comment by webcrumbs-community** [comment]
That's great, @ShatilKhan! Let us know if you need any help (here or at our [discord](https://discord.com/invite/ymVgdPe6rK)).

**Comment by dew-dr0p** [comment]
HI @webcrumbs-community 
Is this issue still opened? Any progress on it? Would like to get it done.

**Comment by webcrumbs-community** [comment]
Hi, @dew-dr0p! You're help is always very welcome. Let's let @ShatilKhan tell us the status of this and if we can assist him with anything.

**Comment by vinayBhoure** [comment]
@webcrumbs-community, which type of error message should we work on? Should it be in the form of a toast or just another popup? Please clarify for better implementation

**Comment by dew-dr0p** [comment]
> Hi, @dew-dr0p! You're help is always very welcome. Let's let @ShatilKhan tell us the status of this and if we can assist him with anything.

Great advise. @ShatilKhan any way we can contribute or help with your work so far on this issue?

**Comment by webcrumbs-community** [comment]
> @webcrumbs-community, which type of error message should we work on? Should it be in the form of a toast or just another popup? Please clarify for better implementation

I'm thinking about those error messages that usually appear in login pages. It could be something like this, e.g.

![image](https://github.com/webcrumbs-community/webcrumbs/assets/116091091/52a630dc-3d51-466b-8aa4-0fadcb4d90d5)

![image](https://github.com/webcrumbs-community/webcrumbs/assets/116091091/e7038480-d3e8-4a8e-b2fd-858f0d67c2c8)

**Comment by ShatilKhan** [comment]
Hey guys sorry for the delay
I'm looking into it

**Comment by dew-dr0p** [comment]
Anyway we could help out, would be really appreciated too. @ShatilKhan

**Comment by Rishav-264** [comment]
Hello @webcrumbs-community I went through the description and I would like to contribute to the project by performing this task.

**Comment by webcrumbs-community** [comment]
Hey, @Rishav-264 and @dew-dr0p. Thanks for offering to take this task! @ShatilKhan had time to check it out, but I believe he got caught up with other things. @dew-dr0p, you offered first, so I'm asking you if you'd like to take this or with you're busy with other tasks and would like @Rishav-264 to help/take. Let us know! Thanks everyone!

**Comment by Rishav-264** [comment]
@webcrumbs-community yes absolutely! Let me know if I am assigned to this task. I have joined the webcrumbs discord server as well so I'll be available there too.

**Comment by dew-dr0p** [comment]
> Hey, @Rishav-264 and @dew-dr0p. Thanks for offering to take this task! @ShatilKhan had time to check it out, but I believe he got caught up with other things. @dew-dr0p, you offered first, so I'm asking you if you'd like to take this or with you're busy with other tasks and would like @Rishav-264 to help/take. Let us know! Thanks everyone!

Hi @webcrumbs-community, there's no problem @Rishav-264 should take up this one, while we still try to close up the one I'm already on and I'll also like to pick up the one on Module Federation.

**Comment by Rishav-264** [comment]
@dew-dr0p works out, @webcrumbs-community let me know when I can start with this.

**Comment by webcrumbs-community** [comment]
Go for it, @Rishav-264 ! Let us know if you need any help. I'll assign this for you

**Comment by Rishav-264** [comment]
@webcrumbs-community Awesome! Will get started with it.

**Comment by dew-dr0p** [comment]
@Rishav-264 Heyyy, well done. How's it going with this issue? How far have you gone? Do you need any help? Pls, remember don't forget to reach out when you need help. I'll be expecting some feedback on how it's going and how far you've gone. 
Have a great time, working on this.

**Comment by Rishav-264** [comment]
> @Rishav-264 Heyyy, well done. How's it going with this issue? How far have you gone? Do you need any help? Pls, remember don't forget to reach out when you need help. I'll be expecting some feedback on how it's going and how far you've gone. 
> Have a great time, working on this.

Hey @dew-dr0p, thanks for helping me out with the problem I was having. It's going well, give me a few days and I'll update you with a demo update. Thanks for your patience
